scripts/variantstore/README.md

```diff
@@ -1,9 +1,31 @@
-
-
+# Running the Genome Variant Store (GVS) Pipeline for AoU
+
+## Steps
+1. If this is the first time running the GVS pipeline in a particular Google billing project, create a support ticket for the BigQuery team that includes "enabling cluster metadata pruning support for the BQ Read API."
```

Comment:
Can you include a link or email address for creating that support ticket? Or maybe that's obvious to people who use BQ more regularly than me :)

Reply:
Used more verbiage from Randi's email
> use your GCP account team to create a support ticket